src/localization/online_localizer/online_localizer.h

```diff
@@ -85,23 +74,23 @@ class OnlineLocalizer {
   void visualize() const;
 
 private:
-  int _querySize = 0;
-  int _slidingWindowSize = 5; // frames
-  bool _needReloc = false;
-  double _expansionRate = -1.0;
-  double _nonMatchCost = -1.0;
+  const int kSlidingWindowSize_ = 5; // frames
```

Comment:
I'm not sure if you want to use `const` here. It makes your class non-copyable. If you want this variable to never be changed and to be the same for all instances of this class, it should probably be an: 
```c++
inline static constexpr int kSlidingWindowSize_ = 5;
```
If you want this number to be different for different instances of the class, then you might want to remove `const` altogether and just make sure it does not change using the logic stored in your class.